### PR TITLE
Completed CSS Positioning - Amelia Talkington

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-01-22 at 11 19 04 AM
Screen Shot 2021-01-22 at 11 39 29 AM
Screen Shot 2021-01-22 at 11 37 57 AM

### CODE REVIEW

File: css-positioning/relative.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+#pikachu {
+  position: relative;
+  top: 240px;
+}
+
+#jigglypuff {
+  position: relative;
+  top: 400px;
+}
```
Comment (codingcodymiller) [inline css-positioning/relative.css:36]: You can remove `position: relative` from these, since you already applied it in the `.pokemon > img` rule above

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! One minor change:

**Comment by ameliatalkington** [comment]
same output -- removed excess code

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**